src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+const lis = tree.querySelectorAll('li');
```

Comment:
```suggestion
const list = tree.querySelectorAll('li');
```
Use full, informative names